database/database.sql

```diff
@@ -2038,6 +2038,18 @@ CREATE TABLE IF NOT EXISTS `d_user_score` (
   CONSTRAINT `FK_d_user_score_d_user` FOREIGN KEY (`user`) REFERENCES `d_user` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
 
+DROP TABLE IF EXISTS `d_history`;
+CREATE TABLE `d_history`
+(
+    `season`      int(11) NOT NULL,
+    `tick`        int(11) NOT NULL,
+    `reports`     longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin DEFAULT NULL CHECK (json_valid(`reports`)),
```

Comment:
why nullable?

Reply:
explained in the following comments, removed (apart from reports which need to be solved later)